administrator/language/en-GB/com_finder.ini

```
@@ -108,7 +109,7 @@ COM_FINDER_HEADING_PHRASE="Search Phrase"
 COM_FINDER_HEADING_RESULTS="Results"
 COM_FINDER_HEADING_SEARCH_TERM_ASC="Search Phrase ascending"
 COM_FINDER_HEADING_SEARCH_TERM_DESC="Search Phrase descending"
-COM_FINDER_INDEX="Index"
+COM_FINDER_INDEX="Start the indexer"
```

```suggestion
COM_FINDER_INDEX="Index"
```
Revert for now to merge PR. Will do a separate PR for the button renaming.